### ISSUE TITLE
Ace reports missing metadata even when it's present

### ISSUE THREAD

**Issue by marisademeglio** [daisy/ace#145]
Accessibility metadata reported as missing in this book, but it's present:
https://github.com/daisy/ace/tree/master/tests/data/base-epub-30

**Comment by marisademeglio** [comment]
This behavior no longer occurs; not sure what changed to affect it.
